docker-compose.yml

```
@@ -1,107 +1,106 @@
 services:
-    app:
-        build: ./
-        volumes:
-            - ./application:/data
-        env_file:
-            - ./local.env
+  app:
+    build: ./
+    volumes:
+      - ./application:/data
+    env_file:
+      - ./local.env
 
-    broker:
-        image: silintl/idp-id-broker:develop
-        platform: linux/amd64
-        environment:
-            API_ACCESS_KEYS: local-sync-123
-            EMAIL_SERVICE_accessToken: dummy
-            EMAIL_SERVICE_assertValidIp: "false"
-            EMAIL_SERVICE_baseUrl: http://email
-            EMAIL_SERVICE_validIpRanges: 10.20.40.0/24
-            EMAIL_SIGNATURE: Dummy Signature for Local Development
-            EMAILER_CLASS: \Sil\SilIdBroker\Behat\Context\fakes\FakeEmailer
-            IDP_NAME: local
-            MYSQL_ROOT_PASSWORD: rootpass
-            MYSQL_HOST: brokerdb
-            MYSQL_DATABASE: broker
-            MYSQL_USER: broker
-            MYSQL_PASSWORD: broker
-            PASSWORD_FORGOT_URL: https://www.example.com/forgot
-            PASSWORD_PROFILE_URL: https://www.example.com/profile
-            SUPPORT_EMAIL: support@example.com
-            MFA_TOTP_apiBaseUrl: not_needed_here
-            MFA_TOTP_apiKey: not_needed_here
-            MFA_TOTP_apiSecret: not_needed_here
-            MFA_WEBAUTHN_apiBaseUrl: not_needed_here
-            MFA_WEBAUTHN_apiKey: not_needed_here
-            MFA_WEBAUTHN_apiSecret: not_needed_here
-        ports:
-            - "8081:80"
-        command: whenavail brokerdb 3306 60 ./run.sh
+  broker:
+    image: silintl/idp-id-broker:latest
```

Maybe a major version would be better, to catch breaking changes.
```suggestion
    image: silintl/idp-id-broker:7
```